lib/internal/abort_controller.js

```diff
@@ -41,12 +41,11 @@ const {
 } = require('internal/errors');
 const {
   converters,
+  createInterfaceConverter,
   createSequenceConverter,
 } = require('internal/webidl');
 
 const {
-  validateAbortSignal,
```

Comment:
Shouldn't we update `validateAbortSignal` – or remove it if we no longer needs it?

Reply:
I tried updating it, but it breaks some tests, as it is used variously in other modules. Some modules treat `EventTarget` with `{ aborted: boolean; }` as a valid `AbortSignal`, and its test is written in that way.